codegen/aws/core/src/main/java/software/amazon/smithy/python/aws/codegen/AwsPythonDependency.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package software.amazon.smithy.python.aws.codegen;
+
+import software.amazon.smithy.python.codegen.PythonDependency;
+import software.amazon.smithy.utils.SmithyUnstableApi;
+
+/**
+ * AWS Dependencies used in the smithy python generator.
+ */
+@SmithyUnstableApi
+public class AwsPythonDependency {
+    /**
+     * The core aws smithy runtime python package.
+     *
+     * <p>While in development this will use the develop branch.
+     */
+    public static final PythonDependency SMITHY_AWS_CORE = new PythonDependency(
```

Comment:
SMITHY_AWS_CORE is [referenced later](https://github.com/SamRemis/smithy-python/pull/1/files#diff-12cb812e3528867ff119f3f7bd0ab5874ae9f1e164fd99769b28bb2127c91d51R44) - that will end up putting this in the pyproject.toml.